src/controllers/playback/video/index.js

```diff
@@ -1407,11 +1409,26 @@ export default function (view) {
             chapterThumbTextContainer.classList.add('chapterThumbTextContainer');
             chapterThumbContainer.appendChild(chapterThumbTextContainer);
 
+            chapterThumbName = document.createElement('div');
+            chapterThumbName.classList.add('chapterThumbText', 'chapterThumbText-dim');
+            chapterThumbTextContainer.appendChild(chapterThumbName);
+
             chapterThumbText = document.createElement('h2');
             chapterThumbText.classList.add('chapterThumbText');
             chapterThumbTextContainer.appendChild(chapterThumbText);
         }
 
+        let chapter;
+        if (item?.Chapters?.length) {
+            for (let i = 0, length = item.Chapters.length; i < length; i++) {
+                const currentChapter = item.Chapters[i];
+
+                if (positionTicks >= currentChapter.StartPositionTicks) {
+                    chapter = currentChapter;
+                }
+            }
+        }
```

Comment:
Not tested.
```suggestion
        for (const currentChapter of item.Chapters || []) {
            if (positionTicks < currentChapter.StartPositionTicks) {
                break;
            }

            chapter = currentChapter;
        }
```
`item.Id` is accessed below, so I think we can skip testing it (`?.`).
I also think we can stop the loop in the `else` block, assuming the chapters are sorted by time. I have rearranged the loop body. We can use `findLast` instead of loop though.